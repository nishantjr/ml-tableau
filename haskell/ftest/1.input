symbols C:0 S:1 Z:123;

--- Sample w/parens explicit; we should parse this and the no-parens
    version the same. ;
--- sat  (μ X . ( (z ∨ (<s X> ∧ y)) ∨ ([f z] ∧ (∃ x. [g ∀ q. <f <g z>>])) ));

--- Add comments at level of symbols/sat/etc, must end with semicolon. ;
--- μ X. ∃ y. … ;
--- ∃ y. μ X. … ;
