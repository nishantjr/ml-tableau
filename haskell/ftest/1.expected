fromList [("C",0),("S",1)]
